doc/construct_template.rst

```diff
@@ -0,0 +1,55 @@
+======================================
+Constructing a New Wavelength Template
+======================================
+
+.. index:: construct_template
+
+Overview
+========
+
+This doc describes how to make the wavelength calibration used in
+`Automated Algorithms`_
+
+See :doc:`wave_calib` for a discussion of various algorithms and
+see :doc:`master_wvcalib` for a discussion of the
+main outputs and good/bad examples.
+
+Finding Data
+============
+
+To build a new template, the requirements are wavelength calibrations for the
+arc spectra over the full wavelength range needed. This may take multiple
+exposures of arc lamps to cover the full range.
+
+Using the Holy Grail Algorithm
+==============================
+
+Once a set of different arc lamp exposures that cover the desired range,
```

Comment:
Once **you have** a set of different arc lamp exposures that cover the desired range